clusterware-customize/lib/functions/customize-repository.functions.sh

```diff
@@ -244,6 +244,8 @@ customize_repository_apply() {
       if [[ "$is_preinit" != "preinit" ]]; then
         echo "Running configure for $profile_name"
         customize_run_hooks "configure:$repo_name-$profile_name"
+        customize_run_hooks "start:$repo_name-$profile_name"
+        customize_run_hooks "node-start:$repo_name-$profile_name"
```

Comment:
This should be `node-started`.

The `echo` line should probably be modified to say "Running event hooks for..." or something to match the new behaviour.